filebeat/module/mysql/slowlog/_meta/fields.yml

```diff
@@ -31,6 +35,55 @@
       type: long
       description: >
         The connection ID for the query.
+    - name: thread_id
```

Comment:
This should be mapped against ECS.

Reply:
I'll review it, I think this is actually the same thing as the connection ID but it appears in different places depending on the version.

Reply:
@ruflin I have been checking and thread, process, and connection ids used by mysql are specific of mysql, they cannot be mapped to process or thread ids in the operating system. 

For the ids in the error log I have also seen big and apparently random numbers that I couldn't map with any other of the IDs.

Should we still use `process.thread.id` even if they are mysql-specific ids?

Reply:
(these big ids only happen in mariadb 10.1 and 10.2, not in other versions of mariadb or mysql, this might be some bug)

Reply:
if the values do not correlate to other ids, I don't think we should map it.

Reply:
I am going to use connection id also for this thread id by now, as they are the same. And I'll open a follow up to discuss if ids of both filesets (and potentially metricsets) should be placed in the same `mysql.thread_id`. So we can at least correlate them along mysql events.